raphtory/src/python/graph/graph.rs

```diff
@@ -366,6 +354,13 @@ impl PyGraph {
         Ok(PyBytes::new(py, &bytes))
     }
 
+    /// Creates a graph from a bincode encoded graph
```

Comment:
I think we can probably remove this now we have the new protobuf storage. Possibly also an option that we add a download_graph function on the client which does this internally?

Reply:
g = client.receive_graph("user/shivam/g3")

-- not to do now